plugins/source/aws/resources/services/ec2/key_pairs.go

```diff
@@ -4,23 +4,29 @@ import (
 	"context"
 
 	"github.com/aws/aws-sdk-go-v2/service/ec2"
+	"github.com/aws/aws-sdk-go-v2/service/ec2/types"
 	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
 	"github.com/cloudquery/cq-provider-sdk/provider/diag"
 	"github.com/cloudquery/cq-provider-sdk/provider/schema"
 )
 
 //go:generate cq-gen --resource key_pairs --config gen.hcl --output .
-func Ec2KeyPairs() *schema.Table {
+func KeyPairs() *schema.Table {
 	return &schema.Table{
-		Name:          "aws_ec2_key_pairs",
-		Description:   "Describes an EC2 Key Pair.",
-		Resolver:      fetchEc2KeyPairs,
-		Multiplex:     client.ServiceAccountRegionMultiplexer("ec2"),
-		IgnoreError:   client.IgnoreAccessDeniedServiceDisabled,
-		DeleteFilter:  client.DeleteAccountRegionFilter,
-		Options:       schema.TableCreationOptions{PrimaryKeys: []string{"key_pair_id"}},
-		IgnoreInTests: true,
+		Name:         "aws_ec2_key_pairs",
+		Description:  "Describes an EC2 Key Pair.",
+		Resolver:     fetchEc2KeyPairs,
+		Multiplex:    client.ServiceAccountRegionMultiplexer("ec2"),
+		IgnoreError:  client.IgnoreAccessDeniedServiceDisabled,
+		DeleteFilter: client.DeleteAccountRegionFilter,
+		Options:      schema.TableCreationOptions{PrimaryKeys: []string{"arn"}},
```

Comment:
Made `arn` primary key. The `arn` is based on the key name and one cannot create duplicate names (I checked)